media/js/mozorg/home/ga-tracking.js

```diff
@@ -5,6 +5,15 @@
 $(function () {
     'use strict';
 
+    var $mainNav;
+
+    //generic links
+    $mainNav = $('#nav-main-menu li a');
+    $mainNav.attr({
+        'data-element-location': 'nav click',
+        'data-link-type': $mainNav.data('name')
+    });
+
```

Comment:
This was recently refactored out in #3287 as it was sending incorrect data to GTM. The attributes are handled now directly in the template, so doesn't need JS iterating over things again.